wandb/sdk/wandb_run.py

```diff
@@ -923,6 +946,23 @@ def finish(self, exit_code=None):
     def join(self, exit_code=None):
         self.finish(exit_code=exit_code)
 
+    def plot_table(self, vega_spec_name, data_table, fields, string_fields=None):
+        """Creates a custom plot on a table.
+        Args:
+            vega_spec_name: the name of the spec for the plot
+            table_key: the key used to log the data table
```

Comment:
is the table_key arg outdated?

Reply:
I'm going to update the design doc to reflect the final status of this.